index.js

```diff
@@ -141,15 +178,26 @@ const line = []
 
 
 // *** Uncomment the lines below to test
-// console.log("%cScope & Closures - Question 1", "color: red")
+console.log("%cScope & Closures - Question 1", "color: red")
 
-// console.log(takeATicketNumber(line))
+function takeATicketNumber(line) {
+  
+  for(let i = 0; i < line.length; i++ ){
+    line.push(`${i+1}`);
+  }
+
+  return `Welcome. You are ticket number ${line}`
+  
+  
+}
```

Comment:
Hmm, whenever we call this function, we should only be updating the ticket number by one and pushing one ticket to the `line` array. Take another jab and then check out the solution and see if you can trace what the code is doing